src/Findologic/Request/FindologicRequestFactory.php

```diff
@@ -74,6 +74,9 @@ protected function setDefaults(
         $searchNavigationRequest->setUserIp($this->fetchClientIp());
         $searchNavigationRequest->setRevision($this->getPluginVersion());
         $searchNavigationRequest->setOutputAdapter(OutputAdapter::XML_21);
+        $searchNavigationRequest->addIndividualParam('shopType', 'Shopware6', true);
+        $searchNavigationRequest->addIndividualParam('shopVersion', Utils::getCleanShopwareVersion(), true);
```

Comment:
How can this work? The third parameter of this method is `$method` and it expects a `string`.

![image](https://user-images.githubusercontent.com/20555112/144216140-5341ff41-945d-47bd-939e-c7a2569233fd.png)